Dockerfile

```
@@ -1,19 +1,21 @@
-FROM node:lts-alpine
+FROM node:lts-alpine AS build-step
 
 ARG catalogURL
 
-RUN npm install -g serve
-
 WORKDIR /app
-
 COPY package*.json ./
-
 RUN npm install
-
 COPY . .
-
 RUN npm run build -- --catalogUrl=$catalogURL
 
+FROM nginx:alpine-slim
```

Same thing here. Specifying the version might save some headaches later on. Also, not sure if this is the latest version.

```
FROM nginx:1.21-alpine
```

---

I'm not sure I agree here. Isn't that a security risk if you don't pick up new versions? Maybe just stick to v1, but accept any other updates? I don't expect that much breaks here, we just serve normal stuff here.

```suggestion
FROM nginx:1-alpine
```